app/parsers/bulkrax/bagit_parser.rb

```diff
@@ -140,7 +140,7 @@ def write_files
           file.write(io.read)
           file.close
           begin
-            bag.add_file(file_name, file.path)
+            bag.add_file(file_name, file.path) if bag.bag_files.select { |b| b.include?(file_name) }.blank?
```

Comment:
because of the work I was doing on https://github.com/samvera-labs/bulkrax/pull/596 on this tenant, I happened to import duplicate files. (known issue on that branch) that work is on hold and we won't come across it in live apps that are using bulkrax.

however, figured we'd put the guard back since it doesn't hurt. and won't throw off our dev testing in the meantime.

Reply:
@alishaevn if this is a temporary guard clause, should we add a `TODO:` to remove it later?

Reply:
we've put this clause up and removed it about 3 times over the last 2 months. since we keep coming across use cases for it...I wasn't sure if it should be temporary, or just leave it here since it doesn't mess anything up either.

each time we find a use case, we find a way around it too so it _could_ be temporary. and currently, I'm the only one who will have the problem unless someone else works on the linked pr above. but leaving it here isn't problematic either.

I'm fine either way.